samples/uikit/build.gradle.kts

```diff
@@ -0,0 +1,162 @@
+import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
+import org.jetbrains.kotlin.gradle.plugin.mpp.NativeBuildType
+
+plugins {
+    kotlin("multiplatform")
+}
+
+allprojects {
+    repositories {
+        jcenter()
+        maven("https://dl.bintray.com/kotlin/kotlin-eap")
+        maven("https://dl.bintray.com/kotlin/kotlin-dev")
+    }
+}
+
+val sdkName: String? = System.getenv("SDK_NAME")
+
+enum class Target(val simulator: Boolean, val key: String) {
+    WATCHOS_X86(true, "watchos"), WATCHOS_ARM64(false, "watchos"),
+    IOS_X64(true, "ios"), IOS_ARM64(false, "ios")
+}
+
+val target = sdkName.orEmpty().let {
+    when {
+        it.startsWith("iphoneos") -> Target.IOS_ARM64
+        it.startsWith("iphonesimulator") -> Target.IOS_X64
+        it.startsWith("watchos") -> Target.WATCHOS_ARM64
+        it.startsWith("watchsimulator") -> Target.WATCHOS_X86
+        else -> Target.WATCHOS_X86
+    }
+}
+
+val buildType = System.getenv("CONFIGURATION")?.let {
+    NativeBuildType.valueOf(it.toUpperCase())
+} ?: NativeBuildType.DEBUG
+
+kotlin {
+    // Declare a target.
+    // We declare only one target (either arm64 or x64)
+    // to workaround lack of common platform libraries
+    // for both device and simulator.
+    val ios = if (!target.simulator ) {
+        // Device.
+        iosArm64("ios")
+    } else {
+        // Simulator.
+        iosX64("ios")
+    }
+
+    val watchos = if (!target.simulator) {
```

Comment:
Do we really need the watchOS target here? The Xcode project defined by the `project.yml` doesn't include it.

Reply:
While technically - no, we may want to have it here as a showcase of how multiple Applete targets could be handled. But actually it belongs to `AppleTargets` plugin, it seems.